lib/rhc/ssh_helpers.rb

```diff
@@ -306,15 +308,20 @@ def restore_snapshot(app, filename, ssh_executable=nil)
       ssh_uri = URI.parse(app.ssh_url)
       ssh_executable = check_ssh_executable! options.ssh
 
-      ssh_cmd = "cat '#{filename}' | #{ssh_executable} #{ssh_uri.user}@#{ssh_uri.host} 'restore#{include_git ? ' INCLUDE_GIT' : ''}'"
-      ssh_stderr = " 2>/dev/null"
-      debug ssh_cmd
```

Comment:
`ssh_cmd` and `ssh_stderr` assignments are still present. I'll re-add the debug statement, I didn't intend to remove it.